healthcareai/common/model_eval.py

```diff
@@ -37,140 +51,122 @@ def clfreport(modeltype, debug, devcheck, algo, X_train, y_train, X_test, y_test
     # Initialize conditional vars that depend on ifelse to avoid PC warning
     y_pred_class = None
     y_pred = None
+    clf = algo
 
     # compare algorithms
-    if devcheck == 'yesdev':
+    if develop_model_mode is True:
         if tune:
-            clf = GridSearchCV(algo,
-                               param,
-                               cv=5,
-                               scoring='roc_auc',
-                               n_jobs=cores)
-        else:
-            clf = algo
+            # Set up grid search
+            clf = GridSearchCV(algo, param, cv=5, scoring='roc_auc', n_jobs=cores)
```

Comment:
1. We are not in the else here. We are in tune=True
2. We are still going to be using randomized.

It bears mentioning that is not the final code. Model eval has to be deconstructed before it can be rebuilt. This is one of many intermediary steps.